wrappers/winuser.go

```diff
@@ -125,6 +228,24 @@ func CloseDesktop(desktop syscall.Handle) error {
 	return nil
 }
 
+func EnumDisplayDevices(device *uint16, devNum uint32, displayDevice *DISPLAY_DEVICE, flags uint32) bool {
+	r1, _, _ := syscall.Syscall6(
+		procEnumDisplayDevices.Addr(),
+		4,
+		uintptr(unsafe.Pointer(device)),
+		uintptr(devNum),
+		uintptr(unsafe.Pointer(displayDevice)),
+		uintptr(flags),
+		0,
+		0)
+	return r1 != 0
+}
+
+func GetSystemMetrics(index uint32) int {
```

Comment:
The type of `index` should be `int32` to match what Microsoft does.  See https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-getsystemmetrics.

Reply:
You are right.